libs/wingsdk/src/util/util.ts

```diff
@@ -97,6 +115,33 @@ export class Util {
     return false;
   }
 
+  /**
+   * Computes the SHA256 hash of the given data.
+   * @param data - The string to be hashed.
+   */
+  public static sha256(data: string): string {
+    return createHash("sha256").update(data).digest("hex"); //SHA256(data).toString();
+  }
+
+  /**
+   * Generates a version 4 UUID.
+   */
+  public static uuidv4(): string {
+    return v4();
+  }
+
+  /**
+   * Generates a unique ID using the nanoid library.
+   * @param options - Optional options object for generating the ID.
+   */
```

Comment:
Adding a link of some kind would be useful here for quick reference

```suggestion
  /**
   * Generates a unique ID using the nanoid library.
   # @link https://github.com/ai/nanoid
   * @param options - Optional options object for generating the ID.
   */
```